src/core/public/styles/core_app/README.txt

```diff
@@ -0,0 +1 @@
+These files are only used by the sass loader, located here: `packages/kbn-optimizer/src/worker/webpack.config.ts`
```

Comment:
Added this file as a reminder, given I lost quite a bit of time finding where these sass files were really used from. I can delete it from the PR if we want.